code/game/gamemodes/roleset/simple.dm

```diff
@@ -50,12 +50,28 @@
 	req_crew = 10
 	event_pools = list(EVENT_LEVEL_ROLESET = -30) //This is an antitag, it has a negative cost to allow more antags to exist
 
+/datum/storyevent/roleset/marshal/can_trigger(var/severity, var/report)
+	var/a_count = 0
+	for(var/datum/antagonist/A in current_antags)
+		if(A.owner && A.is_active() && !A.is_dead())
```

Comment:
`is_active()` include `is_dead()` check.
And probably should include `owner` check (but can missed it right now)